gstats/gstats.c

```diff
@@ -215,7 +215,7 @@ char *GenerateAuthW(const char* challenge, const unsigned short *password, char
 int InitStatsAsync(int theGamePort, gsi_time theInitTimeout)
 {
 	struct sockaddr_in saddr;
-	char tempHostname[128];
+	char tempHostname[sizeof(gcd_gamename)/sizeof(gcd_gamename[0])];
```

Comment:
256+64+2 (gcd_gamename is not enough)

Reply:
https://github.com/GameProgressive/UniSpySDK/pull/27/commits/7fd1270ff84075697b4514d57da20aeffc5174dd